### PR TITLE
Header

### PR DESCRIPTION
**Pull request by carodriguezp**
header component and test
Please review

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { Route, Routes } from "react-router-dom";
 import { Container } from "./App.style";
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (davepadrino) [inline src/App.tsx:3]: Do we really need to import this?

File: src/components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              <span className="icon-bar"></span>
+              <span className="icon-bar"></span>
+            </button>
+
+            <h1 className="navbar-brand">Fruityvice</h1>
+          </div>
+          <div className="collapse navbar-collapse header-styled" id="myNavbar">
+            <ul className="nav navbar-nav">
+              <li className="active">
+                <Link to={`/`}>Discover Fruit</Link>
```
Comment (davepadrino) [inline src/components/header/Header.tsx]: What if we put the routes as variables in an object?

Such as
```
Const routes = {
Base: "/", 
Fruits: "/fruits", 
Etc
} 
```

File: src/components/header/__test__/Header.test.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// describe("",()=>{})
```
Comment (davepadrino) [inline src/components/header/__test__/Header.test.tsx]: Remove. Comment

**Review by davepadrino — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (carodriguezp) [reply src/App.tsx:3]: Yes, we need it

**Pull request merged**